### PR TITLE
Add support for OAuth2 scope parameter

### LINKED ISSUE #198: Oauth2: authorization doesn't have scope parameter
Some of Oauth2 authorization servers(like Google) has *scope* parameter as required, can you please include possibility to specify scope parameter in the OAuthRemoteAppWithRefresh function:
Current:
```
auth = OAuthRemoteAppWithRefresh(
    oauth,
    'auth',
    request_token_url=None,
    access_token_method='POST',
    access_token_url=os.getenv('ACCESS_TOKEN_URL'),
    authorize_url=AUTHORIZE_URL
)
```

To something like:
```
auth = OAuthRemoteAppWithRefresh(
    oauth,
    'auth',
    request_token_url=None,
    access_token_method='POST',
    access_token_url=os.getenv('ACCESS_TOKEN_URL'),
    authorize_url=AUTHORIZE_URL,
    request_token_params = {'scope': os.getenv('SCOPE')}
)
```
Scope example:
{'scope': 'https://www.googleapis.com/auth/gmail.readonly'}

### PR DESCRIPTION
**Pull request by jarik2995**
Added support for scope and update docs. fixes #198

### CODE REVIEW

File: kube_ops_view/main.py [hunk 1]
```diff
@@ -45,7 +45,8 @@
     request_token_url=None,
     access_token_method='POST',
     access_token_url=os.getenv('ACCESS_TOKEN_URL'),
-    authorize_url=AUTHORIZE_URL
+    authorize_url=AUTHORIZE_URL,
+    request_token_params={'scope': os.getenv('SCOPE')}
```
Comment (hjacobs) [inline kube_ops_view/main.py]: Should we make the `request_token_params` option if no scope is given? That would make the change fully backwards-compatible (there might be a difference between sending no params and sending an empty `scope` param).

**Comment by hjacobs** [comment]
Thanks for the PR! See my comment.

File: kube_ops_view/main.py (hunk 1 above)
Comment (jarik2995) [reply kube_ops_view/main.py]: Good idea, probably there might be issues with some vendors if we pass scope as None in request_token_params. I have committed changes to make the request_token_params option None if no scope is given. 
ee256bca7292a0039140579ed48883ecf1829529

**Comment by hjacobs** [comment]
Thanks again :smile:

**Pull request merged**